Fabric.Identity.API/scripts/Install-Identity-Windows.ps1

```diff
@@ -549,19 +549,23 @@ $body = @'
 Write-Console "Registering Fabric.Identity Registration API."
 $registrationApiSecret = Add-ApiRegistration -authUrl $identityServerUrl -body $body -accessToken $accessToken
```

Comment:
Do we need to update this call also, to cast to string and then trim?

Reply:
Api secrets were being returned as expected, for some reason only client secrets were acting differently. I don't think it's a bad idea to cast and trim them to be safe though.